src/components/[guild]/RolePlatforms/components/AddRoleRewardModal/components/AddContractCallPanel/components/CreateNftForm/hooks/useCreateNft.ts

```diff
@@ -48,6 +48,8 @@ export const GUILD_REWARD_NFT_FACTORY_ADDRESSES = {
   SEPOLIA: "0xa9e8e62266d449b766d305075248790bdd46facb",
   IOTA: "0x097E05f7a194a30A482CC9616460498980bE79d3",
   SONIC: "0x070cD1FD4de1ed0259871B7d0b85C0e237702691",
+  SOPHON: "0xTODO",
```

Comment:
Let's comment this out, I think it might cause some unexpected issues (e.g. the Sophon chain will show up on the chain picker inside the create NFT form, but users won't be able to deploy an NFT on it) ;)